packages/appkit-utils/src/solana/SolanaHelpersUtils.ts

```diff
@@ -33,26 +33,43 @@ export const SolHelpersUtil = {
     return SolConstantsUtil.DEFAULT_CHAIN
   },
 
-  getChainFromCaip(chains: CaipNetwork[], chainCaipId: string | undefined | null = ':') {
+  isSolanaChainNameSpace({ chainNamespace }: CaipNetwork) {
+    return chainNamespace.startsWith(CommonConstantsUtil.CHAIN.SOLANA)
+  },
+
+  getChainFromCaip({
```

Comment:
We should consider getting rid of this function I believe

Reply:
Yeah removed 🙏